CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Regression which added a UNC path prefix to the working directory on Windows
 - CLI parameters discarded when config is reload
 - Blurred icons in KDE task switcher (alacritty.ico is now high-res)
+- Inverted first cell when selection is off screen
```

Comment:
Always is incorrect, since it does not happen all of the time and I intentionally used `inverted` here so we don't use `select` twice which sounds a bit odd.

The cell is only inverted, so the statement is correct. It has no impact on the content of the clipboard when copying for example. So inverted/selected should be synonyms in this case.

Reply:
I mean that the cell isn't inverted, it's selected. I mean it's inverted for you, because your selection inverts the content, however mine isn't doing it. So I just wanted to correct you here.

Reply:
> I mean it's inverted for you, because your selection inverts the content, however mine isn't doing it. So I just wanted to correct you here.

Oh, you mean because you've got a fixed selection color? I didn't think about that, that's a good point.

Reply:
>Oh, you mean because you've got a fixed selection color?

Yes, we have fixed selection colors ;- ), so `inverted` is misleading.